### PR TITLE
Check validity of regexes before sending them to the frontend + tooltip for invalid regex 

### PR DESCRIPTION
**Pull request by vdpoora**
https://dev.azure.com/factry/Historian/_workitems/edit/24117

### CODE REVIEW

File: src/CustomVariableEditor/MeasurementFilter.tsx [hunk 1]
```diff
@@ -37,9 +44,12 @@ export function MeasurementFilterRow(props: MeasurementFilterProps) {
...
           aria-label={'Filter measurement'}
           labelWidth={20}
-          tooltip={<div>Searches measurement by name</div>}
+          tooltip={<div>Searches measurement by name, to use a regex surround pattern with /</div>}
         >
-          <Input value={props.initialValue?.Keyword} onChange={(e) => onKeywordChange(e)} />
+          <MaybeRegexInput
+            onChange={(value, ok) => onKeywordChange(value, ok)}
+            initialValue={props.initialValue?.Keyword}
+          />
```
Comment (vereecw) [inline src/CustomVariableEditor/MeasurementFilter.tsx:52]: not sure anymore but I thought with the toggle that the '/' were not needed anymore here and that they were added automatically later. Might be misremembering though

**Review by vereecw — CHANGES_REQUESTED** [review]

**Comment by vereecw** [comment]
still does the request sometimes even though the regex is invalid?

**Comment by vdpoora** [comment]
> still does the request sometimes even though the regex is invalid? When you enter an invalid regex for assets/measurement in the regular query editor en manually press the refresh button

Don't think I can easily prevent that, might be a bigger refactor than it is worth

File: src/variable_support.ts [hunk 2]
```diff
@@ -95,6 +103,12 @@ export class VariableSupport extends CustomVariableSupport<DataSource> {
           ...(JSON.parse(JSON.stringify(request.targets[0].filter)) as AssetFilter | undefined),
           ScopedVars: request.scopedVars,
         }
+        console.log(filter)
```
Comment (vereecw) [inline src/variable_support.ts:106]: remove console.log

**Review by vereecw — APPROVED** [review]

**Pull request merged**